SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -328,7 +328,8 @@ void USpatialReceiver::ReceiveActor(Worker_EntityId EntityId)
 			FString URLString = FURL().ToString();
 			URLString += TEXT("?workerAttribute=") + UnrealMetadataComponent->OwnerWorkerAttribute;
 
-			Connection = NetDriver->AcceptNewPlayer(FURL(nullptr, *URLString, TRAVEL_Absolute), true);
+			// TODO: Once we can checkout PlayerController and PlayerState atomically, we can grab the UniqueId and online subsystem type from PlayerState.
+			Connection = NetDriver->AcceptNewPlayer(FURL(nullptr, *URLString, TRAVEL_Absolute), FUniqueNetIdRepl(), FName(), true);
```

Comment:
We could potentially put the UniqueId and PlatformName inside a handwritten schema component and attach that to the PlayerController entity.

Reply:
Tracking Jira please :)